packages/code-du-travail-frontend/pages/_app.js

```diff
@@ -17,6 +17,10 @@ import { A11y } from "../src/a11y";
 // eslint-disable-next-line import/no-extraneous-dependencies
 import "katex/dist/katex.min.css";
 
+// Get tooltips web-component
+import("../src/web-components/tooltip");
+import("../src/web-components/tooltip-cc");
```

Comment:
using `import` function not static `import`s ?

Reply:
It worked fine with a static import but in the end I put the dynamic import back because I have to check for `window` first in order not to have web-components during server side rendering (it looks for the `window` object and screw it all)